tests/expect/golang/nocontext/test/classes.go

```diff
@@ -1788,21 +1788,21 @@ type HttpExampleOrgTestClassObject struct {
     datetimeScalarProp Property[time.Time]
     // A scalar dateTimeStamp property
     datetimestampScalarProp Property[time.Time]
-    // A enum list property
+    // An enum list property
```

Comment:
Looks like you edited the test expected data instead of the actual source. Sorry it's confusing, the expected test results are a little anacronistic and I should probably look at removing them

Reply:
Sorry about that.
Should I revert this please?